tests/selenium/spec/api-tags-spec.js

```diff
@@ -12,7 +12,7 @@ describe('API tags check spec', () => {
 
   it('shows the Beta lifecycle tags', util.itHelper(async () => {
     await docsPage.navigate(util.fixUrl('/docs/api/getting_started/releases-at-okta'));
-    expect(await docsPage.hasBetaTags(), 'expects Beta tag to be present on page').to.be.true;
+    expect(await docsPage.hasBetaTags(), 'expects Beta tag to be present on page').to.be.false;
```

Comment:
@wayneearl-okta - Any idea how this test passed? I was trying to get a failure 😕

Reply:
isnt this the opposite of what we want now?   we want the beta tag to show, which means to.be.true i would think